UI/style.css

```diff
@@ -57,14 +93,18 @@ header li{
 header #logo{
 	float: left;
 }
-header #user{
+header nav{
 	float: right;
 	margin-top: 10px;
 	font-weight: bold;
 }
 header #logo h1{
 	margin: 0;
 }
+
+#attendant{
```

Comment:
In all cases where there is no space between css selector and opening braces, please insert space